packages/util-use-case/lib/enhancers/UseCaseWithLogger.test.mjs

```diff
@@ -18,19 +18,60 @@ test('create UseCase with custom logger', async (t) => {
   const useCase = new UseCaseWithLogger({
     origin,
     log,
-    name
+    name,
   })
 
   const actual = await useCase.run(params)
 
   t.is(actual, result)
 
   t.assert(origin.run.calledOnce)
+  t.assert(log.debug.calledOnce)
 
   t.deepEqual(log.trace.firstCall.args[0], 'Start use case execution')
+
+  const debugPayload = log.debug.firstCall.args[0]
+  t.deepEqual(debugPayload.useCase, name)
```

Comment:
provide more human-readable messages, like `t.deepEqual(debugPayload.useCase, name, 'should pass use case name in log')`

it helps a lot when searching for a reason for a failed test

Reply:
done